.golangci.yml

```diff
@@ -1,31 +1,43 @@
 linters:
   disable-all: true
   enable:
+  - asasalint
   - asciicheck
   - bodyclose
+  - containedctx
+  - decorder
   - depguard
   - dogsled
   - errcheck
+  - errchkjson
+  - execinquery
   - exportloopref
   - gci
   - goconst
   - gocritic
+  - gocyclo
   - godot
   - gofmt
+  - goheader
   - goimports
   - goprintffuncname
   - gosec
   - gosimple
   - govet
+  - grouper
   - importas
   - ineffassign
+  - logrlint
```

Comment:
This is good lint but rigthnow it is pointing to an archived repo in here https://golangci-lint.run/usage/linters/#disabled-by-default, needs to be revisited!

Reply:
Removed as archived, fixed.